packages/itwin/property-grid/src/components/PropertyGridContent.tsx

```diff
@@ -197,3 +205,19 @@ function useFilterer({ showNullValues, filterText }: UseFiltererProps) {
 
   return compositeFilterer;
 }
+
+function useThrottled(func: () => void, timeout: number) {
+  const isThrottledRef = useLatest(false);
+  const funcRef = useLatest(func);
```

Comment:
I don't think we need this ref as `useThrottled` returns new callback on each call.

Reply:
Removed.